src/integrationTest/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerIssueDecisionIT.java

```diff
@@ -0,0 +1,305 @@
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.sptribs.ciccase.model.ApplicantCIC.APPLICANT_CIC;
+import static uk.gov.hmcts.sptribs.ciccase.model.ContactPreferenceType.EMAIL;
+import static uk.gov.hmcts.sptribs.ciccase.model.RepresentativeCIC.REPRESENTATIVE;
+import static uk.gov.hmcts.sptribs.ciccase.model.RespondentCIC.RESPONDENT;
+import static uk.gov.hmcts.sptribs.ciccase.model.SubjectCIC.SUBJECT;
+import static uk.gov.hmcts.sptribs.testutil.TestConstants.ABOUT_TO_START_URL;
+import static uk.gov.hmcts.sptribs.testutil.TestConstants.ABOUT_TO_SUBMIT_URL;
+import static uk.gov.hmcts.sptribs.testutil.TestConstants.AUTHORIZATION;
+import static uk.gov.hmcts.sptribs.testutil.TestConstants.ISSUE_DECISION_MID_EVENT_URL;
+import static uk.gov.hmcts.sptribs.testutil.TestConstants.SERVICE_AUTHORIZATION;
+import static uk.gov.hmcts.sptribs.testutil.TestConstants.SUBMITTED_URL;
+import static uk.gov.hmcts.sptribs.testutil.TestConstants.TEST_AUTHORIZATION_TOKEN;
+import static uk.gov.hmcts.sptribs.testutil.TestDataHelper.callbackRequest;
+import static uk.gov.hmcts.sptribs.testutil.TestDataHelper.caseData;
+import static uk.gov.hmcts.sptribs.testutil.TestEventConstants.CASEWORKER_ISSUE_DECISION;
+import static uk.gov.hmcts.sptribs.testutil.TestResourceUtil.expectedResponse;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureMockMvc
+@ContextConfiguration(initializers = {IdamWireMock.PropertiesInitializer.class})
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+public class CaseWorkerIssueDecisionIT {
+    private static final String CASEWORKER_ISSUE_DECISION_MID_EVENT_RESPONSE =
+        "classpath:responses/caseworker-issue-decision-mid-event-response.json";
+    private static final String CASEWORKER_ISSUE_DECISION_ABOUT_TO_SUBMIT_RESPONSE =
+        "classpath:responses/caseworker-issue-decision-about-to-submit-response.json";
+    private static final String CONFIRMATION_HEADER = "$.confirmation_header";
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @MockBean
+    private WebMvcConfig webMvcConfig;
+
+    @MockBean
+    private IdamService idamService;
+
+    @MockBean
+    private NotificationServiceCIC notificationServiceCIC;
+
+    @MockBean
+    private CaseDataDocumentService caseDataDocumentService;
+
+    @BeforeAll
+    static void setUp() {
+        IdamWireMock.start();
+    }
+
+    @AfterAll
+    static void tearDown() {
+        IdamWireMock.stopAndReset();
+    }
+
+    @Test
+    void shouldSetDecisionSignatureInAboutToStart() throws Exception {
```

Comment:
Same as other PR - maybe we should emphasise the reset or clearing of decision signature in the test name